libs/map/src/lib/actions/map/map-with-actions.ts

```diff
@@ -1,49 +1,46 @@
-import { defaults } from 'ol/interaction';
+import {defaults} from 'ol/interaction';
 import Map from 'ol/Map';
-import { CONTROL_TYPE } from '../../vl-map.model';
-
-export interface VlMapActionOptions {
-    disableEscapeKey?: Boolean;
-    actions?: any;
-    interactions?: any;
-    disableKeyboard?: Boolean;
-    disableRotation?: Boolean;
-    disableMouseWheelZoom?: Boolean;
+import {CONTROL_TYPE} from '../../vl-map.model';
+import {MapOptions} from "ol/PluggableMap";
+
+export interface VlMapWithActionsOptions extends MapOptions {
+    disableEscapeKey: boolean;
+    disableKeyboard: boolean;
+    disableRotation: boolean;
+    disableMouseWheelZoom: boolean;
 }
 
+
 /**
  * Deze map bevat enkel de functionaliteit om de acties te behandelen. Aan het eerste argument van de constructor kan het gebruikelijke object map opties worden weergegeven die ook op de ol.Map worden gezet, samen met een extra parameter 'acties' in dat object. Deze array bevat MapActions.
  * De eerste actie van de array is steeds de default actie en zal ook hierop gezet worden bij bijvoorbeeld een escape toets.
  *
  * Deze kaart regelt dat er maar één actie actief kan staan. Bij het activeren van een andere actie wordt namelijk de huidige actie gedeactiveerd.
  */
+
 export class VlMapWithActions extends Map {
-    private actions: any[];
+    private _actions: any[] = [];
```

Comment:
Dit is een vraag voor UIG-team denk ik, het feit dat het private is. Is er dan nog een _ prefix nodig?

(naar backwards compatibility toe, geen idee of iemand in JS rechtstreeks dit zette)